cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/power/PowerExample.java

```diff
@@ -267,15 +267,24 @@ private Host createPowerHost(final int id) {
         final ResourceProvisioner ramProvisioner = new ResourceProvisionerSimple();
         final ResourceProvisioner bwProvisioner = new ResourceProvisionerSimple();
         final VmScheduler vmScheduler = new VmSchedulerTimeShared();
+        
+        final double hostStartUpDelay = 5;	//Indicates Host Power On Delay
+        final double hostShutDownDelay = 5;	//Indicates Host Power off Delay        
+        final double hostStartUpPower = 5;	//Indicates Host Power On Power
+        final double hostShutDownPower = 5;	//Indicates Host Power off Power
```

Comment:
We need to think about this startup and shutdown power consumption.
Those operations impact the host CPU load. Currently, PowerModelHost class considers that load to calculate power consumption. For better accuracy, power consumption is computed considering a static power amount of watts and a dynamic one according to the CPU load.
This way, we could use the already existing methods in that class to compute the power consumption.
But instead of statically defining the exact power consumption during startup and shutdown, we could define a UtilizationModel for startup and other for shutdown.

Maybe this will complicate the implementation too much. Anyway, I need to take a time to think about it.
We could even address this as a different issue and PR. Let it be the way it is by now. We decide about it before finishing this PR.